examples/catalog/bin/sample_page.dart

```diff
@@ -87,9 +95,11 @@ class SampleInfo {
   // The value of the 'Classes:' comment as a list of class names.
   Iterable<String> get highlightedClasses {
     final String classNames = commentValues['classes'];
-    if (classNames == null)
-      return const <String>[];
-    return classNames.split(',').map<String>((String s) => s.trim()).where((String s) => s.isNotEmpty);
+    if (classNames == null) return const <String>[];
+    return classNames
```

Comment:
Flutter uses two-space indentation

Reply:
would you want 2 space indentation after every dot?

Reply:
Assuming that you mean "before every dot", then yes, since that's consistent with the overall indentation style.
```
 return classNames
   .split(',')
   .map<String>((String s) => s.trim())
   .where((String s) => s.isNotEmpty);
```

Reply:
let me know if tweaking dartfmt to always use two spaces after every dot would be enough to make this formatting acceptable. it would be an easy tweak to test out although fixing all existing dart code would of course be painful.

Reply:
i've tweaked dartfmt to match the indent of 2 before every dot. Take a look at the latest output.

Reply:
On behalf of myself and all other two-space indentation consistency zealots, thank you.